content/contributing/_index.md

```diff
@@ -2,8 +2,10 @@
 title = "Contributing"
 template = "docs.html"
 page_template = "docs.html"
-insert_anchor_links = "right"
+# TODO: Create introduction page.
+# redirect_to = "/contributing/introduction"
 [extra]
+# TODO: Remove this to publish the Contributing book.
```

Comment:
```suggestion
# TODO: Remove both of these lines below to publish the Contributing book.
```
Clarify the wording so no one thinks they only need to remove `status = 'hidden` and understand you need to remove both lines.